### DISCUSSION TITLE
Templater Question

### DISCUSSION THREAD

**Discussion by gcimmarrusti** [BryanJenksCommunity/FAQ#56] [Q&A] [answered]
Hello Bryan,

I was so thrilled with your video.  I was using the old Templater to put in the date in the file and go between yesterday today and tomorrow and inserting quote of the day all in my daily note.  Since the change, of course my old template does not work.  When trying the changes, I went on the developer's website and tried some of the simple code and looked at yours, but neither will work, it literally copies the coded text.  Example: # Daily Journal - <%tp.file.creation_date()%> for the title and <% tp.web.daily\_quote() %> for the quote, but it only puts in the text (e.g. <% tp.web.daily\_quote() %>) and not the quote.  Any suggestions of what I may be doing wrong?

Thanks,

Greg Cimmarrusti

**Comment by tallguyjenks** [comment, marked as answer]
Hey Greg!

check your hotkey bindings, if you're still inserting templates with the core template command it wont work, you need to insert templates with the templater command. It's something I've often done myself 😅️

**Reply by gcimmarrusti** [reply]
Thanks, Bryan, I will check it out.  Once again, I really appreciate your effort that you put into your videos.  Your enthusiasm makes your videos very approachable.  Keep up the great work, just make sure you take time out for yourself and not get overburdened.  Wishing you all the best!

Best regards,

Greg

**Reply by tallguyjenks** [reply]
Thanks as always Greg for your boundless positivity and support :)
